tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -248,10 +253,17 @@ function cleanup_containerd() {
 
 function reset_runtime() {
 	kubectl label node "$NODE_NAME" katacontainers.io/kata-runtime-
-	systemctl daemon-reload
-	systemctl restart "$1"
-	if [ "$1" == "crio" ] || [ "$1" == "containerd" ]; then
-		systemctl restart kubelet
+
+	if [ -x /sbin/openrc-run ]; then
+		if [ "$1" == "crio" ] || [ "$1" == "containerd" ]; then
+			service restart kubelet
+		fi
+	else
+		systemctl daemon-reload
+		systemctl restart "$1"
+		if [ "$1" == "crio" ] || [ "$1" == "containerd" ]; then
+			systemctl restart kubelet
+		fi
```

Comment:
Here we can also take advantage of the function suggested above, and do something like:
```sh
    restart_service "$1"
    if [ "$1" == "crio" ] || [ "$1" == "containerd" ]; then
        restart_service kubelet
    fi
```

WDYT?

Reply:
Good points!